### ISSUE TITLE
Uncaught TypeError: input.size is not a function

### ISSUE THREAD

**Issue by hairetdin** [Devoter/ember-cli-retech-inline-bs-datetimepicker#1]
I get this error when i use `{{retech-datetimepicker date=model.date}}`

DEBUG: -------------------------------
DEBUG: Ember                : 2.8.2
DEBUG: Ember Data           : 2.9.0
DEBUG: jQuery               : 3.1.1
DEBUG: Ember Django Adapter : 1.1.3
DEBUG: Ember Simple Auth    : 1.1.0
DEBUG: -------------------------------

**Comment by Devoter** [comment]
Sorry, I have to add more examples to readme and fix this bug. Right now try this:
```hbs
{{retech-datetimepicker date=model.date inline=true}}
```

**Comment by Devoter** [comment]
I was wrong. You have to fix jQuery version to 2.2.4 (this is a dependence of Bootstrap DateTimePicker).
`bower.json`
```javascript
"dependencies": {
  "jquery": "2.2.4"
}
```

**Comment by Devoter** [comment]
Fixed, Enjoy jQuery 3.1.1
